### ISSUE TITLE
Rec's - Rules for ordering the display of new recommendations

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub-platform-project#150]
As a PM, I want the display of recommendations to follow a logical pattern and order, so that users don't get confused by the way we present information. 

** Feature Acceptance Criteria **
- [x] New rec's appear at the top
For weeks that have new groupings, group intros should come before group calls.
- [x] Week 1 - rec order should be 1) Post in PLG; 2) First Riff Call; 3) Post in Town Square
- [x] Week 4 - rec order should be 1) Post in Capstone; 2) First Capstone Call; 3) Two calls this week

**Comment by jaedoucette** [comment]
"New" is tricky, because we don't currently have a persistent list of recs.

One possibility is use a common data structure for recs, that requires a specified date at which it becomes active. Recs are then sorted by date. Another is to define an ordering relation over types of Recs.

We think this should probably be done after the second rec is made, so that subsequent rec designs can be made to conform to it.

**Comment by brecriffs** [comment]
Seems like if we create functions to generate each recommendation, and we call the functions in a particular order (on the front-end), we can achieve this.

**Comment by brecriffs** [comment]
I am confused about week 4. The Course Connections card says display only 5 weeks into the course.

**Comment by adonahue** [comment]
Good catch @brecriffs - I've updated the card above for the display for week four. Thanks!
